lib/resolveModule.js

```diff
@@ -1,7 +1,14 @@
 'use strict';
 
 const {join, parse, resolve} = require('path');
-const {readFileSync, statSync} = require('fs');
+const {readFileSync, statSync, realpathSync} = require('fs');
+
+const argv = require('minimist')(process.argv.slice(2), {
+  boolean: ['preserve-symlinks'],
+});
+
+const PRESERVE_SYMLINKS = !!argv['preserve-symlinks'] ||
+  process.env.NODE_PRESERVE_SYMLINKS + '' === '1';
```

Comment:
I usually vote for explicit coercions, cause it's easier to figure out what was the initial intention. So would like to suggest to use here smthing like this: 

```js
const PRESERVE_SYMLINKS = Boolean(argv['preserve-symlinks']) ||
  String(process.env.NODE_PRESERVE_SYMLINKS) === '1';
```

Reply:
And looks like that minimist has already done necessary typecasting here :)